flexget/components/notify/notifiers/slack2.py

```diff
@@ -0,0 +1,237 @@
+                        alt_text: 'plants'
+                      fields:
+                        - '*Score*'
+                        - '*Genres*'
+                        - '{{ imdb_score }}'
+                        - "{{ imdb_genres | join(', ') | title }}"
+                        - '*Rating*'
+                        - '*Cast*'
+                        - '{{ imdb_mpaa_rating }}'
+                        - >
+                          {% for key, value in imdb_actors.items() %}{{ value }}{% if not loop.last %}, {% endif %}
+                          {% endfor %}
+                  - image:
+                      url: "{{ tvdb_banner }}"
+                      alt_text: "{{ series_name }} Banner"
+                  - context:
+                      - image:
+                          url: 'https://image.freepik.com/free-photo/red-drawing-pin_1156-445.jpg'
+                          alt_text: 'red pin'
+                      - text: ':round_pushpin:'
+                      - text: 'Task: {{ task }}'
+                    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'web_hook_url': {'type': 'string', 'format': 'uri'},
+            'blocks': {
+                'type': 'array',
+                'minItems': 1,
+                'maxItems': 50,
+                'items': {
+                    'type': 'object',
+                    'properties': {
+                        'section': {
+                            'type': 'object',
+                            'properties': {
+                                'text': {'type': 'string'},
+                                'image': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'url': {'type': 'string', 'format': 'uri'},
+                                        'alt_text': {'type': 'string'},
+                                    },
+                                    'required': ['url', 'alt_text'],
+                                    'additionalProperties': False,
+                                },
+                                'fields': {
+                                    'type': 'array',
+                                    'minItems': 1,
+                                    'maxItems': 10,
+                                    'items': {'type': 'string'}
+                                }
+                            },
+                            'anyOf': [
+                                {'required': ['text']},
+                                {'required': ['fields']}
+                            ],
+                            'additionalProperties': False,
+                        },
+                        'image': {
+                            'type': 'object',
+                            'properties': {
+                                'text': {'type': 'string'},
+                                'image': {
```

Comment:
seems like you can reuse some definitions. You can set those to different python variables or use json schema's own `definitions` attribute